README.md

```diff
@@ -0,0 +1,205 @@
+# edX ESLint config
+
+ESLint config for edX JavaScript code.
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [License](#license)
+3. [Dependencies](#dependencies)
+4. [Usage](#usage)
+5. [The Config](#the-config)
+
+## Overview
+
+In order to standardize and enforce edX's JavaScript coding style across multiple codebases, edX has adopted ESLint and this shared ESLint config. This document describes the edX JavaScript style and its inspirations.
+
+## License
+
+The code in this repository is licensed the Apache 2.0 license unless otherwise
```

Comment:
I checked with the OSS team and some other members of Docs and verified that Apache is the right license for this project.